app/components/provider_interface/application_card_component.rb

```diff
@@ -40,5 +40,15 @@ def recruitment_cycle_text
         "#{year - 1} to #{year}"
       end
     end
+
+    def course_type
+      return 'undergraduate' if undergraduate?
+
+      'postgraduate'
+    end
+
+    def undergraduate?
+      application_choice.current_course.teacher_degree_apprenticeship?
+    end
```

Comment:
How do you feel about raising this into the Course model?
With the ITT Reform work (in the other squad) we'll be adding a field with `undergraduate` or `postgraduate` as the value. It'd be easy to update this in the Course model when it becomes available 👌

Reply:
Yes, I agree. I also move the course type too and it makes sense to be on the course model as well.